### ISSUE TITLE
Clicking vertical scroll slider causes dropdown to close

### ISSUE THREAD

**Issue by zachhorton** [pozil/sfdc-ui-lookup-lwc#41]
Similar to #12 

Using the slider to scroll through results causes the dropdown box to close.
Using Windows 10, Chrome v80
scroll-after-n-items="10"

Issue not present using Microsoft Edge.

**Comment by mbrandt73** [comment]
We have the same problem.

**Comment by pozil** [comment]
Thanks for letting me know.
There's a chance that this an OS/Browser/SLDS issue that's not specific to the component. Not sure I can fix it.

**Comment by pozil** [comment]
Potential workaround with #45 (keyboard support).

**Comment by qlbs** [comment]
It seems to be able to solve with using mousedown and mouseup in listbox div.

```
/* lookup.html */
...
<!-- Result list box start -->
<div
    id="listbox"
    role="listbox"
    onclick={handleComboboxClick}
    onmousedown={handleMouseDown}
    onmouseup={handleMouseUp}
>
...

/* lookup.js */
_cancelBlur = false;
handleBlur() {
    // Prevent action if selection is either not allowed or cancelled
    if (!this.isSelectionAllowed() || this._cancelBlur) {
        return;
    }
    this.hasFocus = false;
}

handleMouseDown(event) {
    const mainButton = 0;
    if (event.button === mainButton) {
        this._cancelBlur = true;
    }
}
handleMouseUp() {
    this._cancelBlur = false;
    // re-focus to text input for the next blur event 
    this.template.querySelector('input').focus();
}
```

Please see salesforce base combobox component for reference.
https://github.com/salesforce/base-components-recipes/tree/master/force-app/main/default/lwc/baseCombobox

**Comment by pozil** [comment]
Thanks for the great suggestion @qlbs and sorry for the delay in getting back to you.

@qlbs, @zachhorton, @mbrandt73 could one of you check out the `pozil/blur-fix` branch and check if the fix works on IE before I merge PR #48 ?

**Comment by pozil** [comment]
Fixed with PR #48
